micro-deps/micro-deps/src/main/groovy/com/ofg/infrastructure/discovery/watcher/DependencyState.java

```diff
@@ -0,0 +1,6 @@
+package com.ofg.infrastructure.discovery.watcher;
+
+enum DependencyState {
```

Comment:
You might be right - my way of thinking was like following:
only this package uses that so I changed to package scope